src/plugins/displayLayout/components/SubobjectView.vue

```diff
@@ -0,0 +1,118 @@
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+<template>
+    <div class="u-contents">
+        <div class="c-so-view__header">
+            <div class="c-so-view__header__start">
+                <div class="c-so-view__name icon-object">{{ item.domainObject.name }}</div>
+                <div class="c-so-view__context-actions c-disclosure-button"></div>
+            </div>
+            <div class="c-so-view__header__end">
+                <div class="c-button icon-expand local-controls--hidden"></div>
+            </div>
+        </div>
+        <object-view class="c-so-view__object-view"
+                     :object="item.domainObject"></object-view>
+    </div>
+</template>
+
+<style lang="scss">
+    @import "~styles/sass-base";
+
+    .c-so-view {
+        /*************************** HEADER */
+        &__header {
+            display: flex;
+            align-items: center;
+            flex: 0 0 auto;
+            margin-bottom: $interiorMargin;
+
+            > [class*="__"] {
+                display: flex;
+                align-items: center;
+            }
+
+            > * + * {
+                margin-left: $interiorMargin;
+            }
+
+            [class*="__start"] {
+                flex: 1 1 auto;
+                overflow: hidden;
+            }
+
+            [class*="__end"] {
+                //justify-content: flex-end;
+                flex: 0 0 auto;
+
+                [class*="button"] {
+                    font-size: 0.7em;
+                }
+            }
+        }
+
+        &__name {
+            @include ellipsize();
+            flex: 0 1 auto;
+            font-size: 1.2em;
+
+            &:before {
+                // Object type icon
+                flex: 0 0 auto;
+                margin-right: $interiorMarginSm;
+            }
+        }
+
+        /*************************** OBJECT VIEW */
+        &__object-view {
+            flex: 1 1 auto;
+            overflow: auto;
+
+            .c-object-view {
+                .u-fills-container {
+                    // Expand component types that fill a container
+                    @include abs();
+                }
+            }
+        }
+    }
+</style>
+
+<script>
+    import ObjectView from '../../../ui/components/layout/ObjectView.vue'
+
+    export default {
+        inject: ['openmct'],
+        props: {
+            item: Object
+        },
+        components: {
+            ObjectView,
+        },
+        mounted() {
+            this.item.config.attachListeners();
```

Comment:
Hrm, this looks funky but not sure if there's better handling available.  But we can address this next sprint w/ display layout drawing.

Reply:
The listeners are defined in the view configuration classes for each type (subobject and telemetry). We can discuss offline.